Gemfile

```
@@ -8,6 +8,7 @@ ruby '3.0.2'
 gem 'rails', '~> 6.1.5'
 
 gem 'bootsnap', '>= 1.4.4', require: false # https://rubygems.org/gems/bootsnap
+gem 'discard', '~> 1.2'
```

Falto la ruta con el link a rubygems de la gema y ya estaría listo.
```suggestion
gem 'discard', '~> 1.2' # gem 'discard', '~> 1.2', '>= 1.2.1'
```